pylint/checkers/similar.py

```diff
@@ -863,8 +857,8 @@ def close(self):
 
             self.add_message("R0801", args=(len(couples), "\n".join(msg)))
             duplicated += num * (len(couples) - 1)
-        stats["nb_duplicated_lines"] = duplicated
-        stats["percent_duplicated_lines"] = total and duplicated * 100.0 / total
+        stats.nb_duplicated_lines += int(duplicated)
+        stats.percent_duplicated_lines += float(total and duplicated * 100.0 / total)
```

Comment:
Unrelated to the change but `total and duplicated` is equivalent to `total + duplicated` ?

Reply:
No, it seems to take `duplicated` whenever `total == 0`. See [docs](https://docs.python.org/3/library/stdtypes.html)